### ISSUE TITLE
A11y - About page

### ISSUE THREAD

**Issue by luistoptal** [gigascience/gigadb-website#1368]
## User story

- Heading levels are skipped: h4 present, missing any higher level heading
- hero image does not have alt attribute
- the text has many llinks to the same url (https://beta.gigadb.org/site/index), consider removing most of them as it can be a bit overwhelming to navigate with keyboard
- There is a list of logos that uses a table as layout, it should use a flex box or some other element (not a table), additionally, the logos don't have an alt attribute so assistive technologies cannot identify them

https://beta.gigadb.org/site/about

## Acceptance criteria

- [x] Do not skip heading levels (h1, h2...) (prio 3)
- [x] Add alt attrbiute to hero image (prio 2)
- [x] Logo list uses something that is not a table for layout (a flex box, probably) (prio 1)
- [x] Logo images have alt attribute (prio 3)

**Comment by luistoptal** [comment]
Working branch: https://github.com/luistoptal/gigadb-website/tree/new-feature/1368/a11y-about-page

**Comment by only1chunts** [comment]
@luistoptal can this ticket be closed?
